### DISCUSSION TITLE
Questions Regarding Superscript in OnlyOffice Editor

### DISCUSSION THREAD

**Discussion by jonjonnz** [ONLYOFFICE/DocumentServer#1811] [Q&A]
What's the object related to Superscript and subscript in OnlyOffice editor
![Screenshot from 2022-06-30 16-45-21](https://user-images.githubusercontent.com/49023172/176664129-c4b84712-357f-49a1-9d51-9566f8613e81.png)

How is the superscript object stored in which paragraph object

**Comment by ShockwaveNN** [comment]
Hi, I don't think I'm qualified to anwser this question so I think you should use debugger in chrome and figure it out yourself
